### DISCUSSION TITLE
LG Fan implementation

### DISCUSSION THREAD

**Discussion by st-karanjamwal** [nVuln/homebridge-lg-thinq#264] [General]
@nVuln thrilled to know about this plugin lately. I started using this plugin on my Hoobs device. I wonder if a new update for LG Fan is coming or not. I will be happy to contribute.

**Comment by nVuln** [comment]
I'm very happy for your contribute, I don't have any fan there, so hard to dev & testing it
waiting for your PR

**Reply by st-karanjamwal** [reply]
Sure thing @nVuln. Let me start on this one. Thank you

**Reply by st-karanjamwal** [reply]
@nVuln I was getting the following error:
zsh: permission denied: thinq
How can I test the fan implementation? Can you provide me with some steps, please?

**Reply by nVuln** [reply]
you need install it to homebridge on your local pc, try run and write a debug log to homebridge debug logging
